app/graphql/mutations/users/create_user.rb

```diff
@@ -0,0 +1,16 @@
+module Mutations
+  module Users
+    class CreateUser < ::Mutations::BaseMutation
+      argument :first_name, String, required: true
+      argument :last_name, String, required: true
+      argument :email, String, required: true
+      argument :google_token, String, required: true
```

Comment:
For phase two, we may add in user's est-time for self care which would be also input here, correct?

Reply:
We have an accessToken, an IDToken and a refreshToken, I'm not too familiar but something for us to dig into to see which ones we want to store. We might need to modify structure accordingly.